### PR TITLE
chore(gemfile): Includes Gemfile.lock

### PR DESCRIPTION
**Pull request by mapreal19**
https://bundler.io/guides/faq.html#committing-lockfiles-in-libraries

- [x] Remove rubinius gems

### CODE REVIEW

File: .rubocop_todo.yml [hunk 1]
```diff
@@ -6,13 +6,6 @@
 # Note that changes in the inspected code, or installation of new
 # versions of RuboCop, may require this file to be generated again.
 
-# Offense count: 1
```
Comment (mapreal19) [inline .rubocop_todo.yml]: one less more to go :)

**Comment by mapreal19** [comment]
I'll try to fix travis if this looks good

**Comment by inukshuk** [comment]
I think it would be OK to remove all the Rubinius specific Gems (that would remove a lot of clutter).

**Comment by mapreal19** [comment]
@inukshuk ready for review. I also ended up updating bundler (we're safe since we don't support ruby < 2.3)

**Comment by inukshuk** [comment]
Look good to me!

**Comment by mapreal19** [comment]
hopefully this will help for future contributors :)

**Pull request merged**